controller/linodemachine_controller.go

```diff
@@ -294,29 +313,13 @@ func (r *LinodeMachineReconciler) reconcileCreate(
 
 		linodeInstance = &linodeInstances[0]
 	case 0:
-		createConfig := linodeMachineSpecToInstanceCreateConfig(machineScope.LinodeMachine.Spec)
-		if createConfig == nil {
-			err = errors.New("failed to convert machine spec to create instance config")
-
-			logger.Error(err, "Panic! Struct of LinodeMachineSpec is different than InstanceCreateOptions")
-
-			return err
-		}
-		createConfig.Label = machineScope.LinodeMachine.Spec.Label
```

Comment:
@AshleyDumaine `linodeMachineSpecToInstanceCreateConfig` set the label, we don't have to set it manually. We have test for it: https://github.com/linode/cluster-api-provider-linode/blob/main/controller/linodemachine_controller_helpers_test.go#L22 so there must be an error is it is empty after the conversion.